projects/core/src/cart/connectors/payment/cart-payment.connector.ts

```diff
@@ -0,0 +1,27 @@
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { PaymentDetails } from '../../../occ/occ-models/occ.models';
+import { CartPaymentAdapter } from './cart-payment.adapter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CartPaymentConnector {
+  constructor(private adapter: CartPaymentAdapter) {}
+
+  public createDetails(
```

Comment:
just `create()` ?

Reply:
Changed `createDetails` to `create` and `setDetails` to `set`,